### DISCUSSION TITLE
Weekly #1

### DISCUSSION THREAD

**Discussion by Ninetilt** [Ninetilt/Project-TDC#3] [General]
### **Vorstellung unseres Teams**
- **Hannah** (_Product Owner / Entwickler_): Sie sorgt dafür, dass die Anforderungen des Projekts klar definiert und priorisiert werden.
- **Christian** (_Scrum-Master / Entwickler_): Unterstützt das Team bei der Einhaltung der Scrum-Prozesse.
- **Julian** (_Blog-Master / Entwickler_): Verantwortlich für die Pflege und Aktualisierung unseres Blogs.
- **Leo** (_Entwickler_): Kümmert sich um die Entwicklung der App.
- **Nico** (_Entwickler_): Kümmert sich um die Entwicklung der App.
   
Zusätzliche Rollen werden evtl. noch später definiert.

### **Vision unseres Projektes**
Unsere Vision ist es, eine innovative App zu liefern, die durch ihre Funktionalität und Stabilität überzeugt. Dabei setzen wir auf bewährte Technologien und agile Entwicklungsmethoden, um eine leistungsstarke und benutzerfreundliche Anwendung zu schaffen. Für genauere Informationen über die Funktionalität unserer App, sieh dir diesen Blog-Post an: [Was ist Project TDC](https://github.com/Ninetilt/Project-TDC/discussions/2).

### **Techniken und Plattformen, die verwendet werden**
Für die Umsetzung unserer App nutzen wir folgende Technologien und Tools:
- **Programmiersprache**: C#
- **Framework**: .NET MAUI für die plattformübergreifende Entwicklung.
- **Entwicklungsumgebung**: Visual Studio
- **Versionsverwaltung**: Git, mit Branches für Main, Feature-Branches usw.
- **Datenbanken und CI/CD**: Docker und Jenkins werden später integriert.
- **Testverfahren**: Unit-Tests und Rest-Tests (ebenfalls in späteren Sprints).

### **Scrum-Setup**

Wir arbeiten nach dem Scrum-Framework und haben unser Setup wie folgt gestaltet:
- Sprintlänge: 2 Wochen
- Meetings:
   - Weekly: Jeden Montag von 20-21:30 Uhr.
   - Retrospektiven: Alle zwei Wochen, um den Sprint zu reflektieren und Verbesserungen zu besprechen.
   - Dailys: Je nach Bedarf, um kurzfristige Abstimmungen zu ermöglichen.
- Plattform: Wir nutzen Jira, um unser Scrum-Board zu pflegen, Backlogs zu verwalten und Aufgaben zu verfolgen.
- Projektbearbeitungszeit: Wird in Story Points gemessen. 1 Story Point = 1h, Abschätzungen für Aufgaben werden in 2-er Potenzschritten (2, 4, 8, 16) gemacht.

### **Sprint 1 Backlog**
![Sprint1Backlog](https://github.com/user-attachments/assets/e4754a50-9264-4440-87cc-73c6eb814e63)

### **Sprint-Board**
![jiraboard](https://github.com/user-attachments/assets/85977e9d-6022-402f-a7f1-09621deb212c)

**Comment by bunki1224** [comment]
Ein wirklich spannendes Projekt! Die Idee, eine To-Do-Liste zu "gamifizieren" und den Nutzern die Möglichkeit zu geben, sich mit Freunden zu messen, ist großartig. Der kompetitive Ansatz motiviert, Aufgaben schneller und vor den Freunden zu erledigen. Besonders positiv finde ich, dass die Rollen klar verteilt und die Aufgaben gut beschrieben sind. Auch die Entscheidung, das Projekt als Android-App umzusetzen, ist sinnvoll und erreicht eine breite Nutzerbasis.

>Mit freundlichen Grüßen,
>das Trip Ahead Team (JB)

**Comment by ldcdorn** [comment]
Wir finden die Idee, die Aspekte von gamification mit einer To-Do-App zu verbinden, sehr spannend. Sind schon darauf gespannt, wie sich die gamification Aspekte in der App konkret bemerkbar machen. C# mit MAUI zu benutzen, um die Platformunabhängigkeit zu gewährleisten, halten wir für super.
Auf welche Platformen wollt ihr denn eure To-Do-App anfangs shippen? Kann man die App dann auch auf Mobileplatformen nutzen?
 
Freundliche Grüße
Project HÆLTH
